test/third_party/install.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env sh
```

Comment:
Should these two files live under third_party/gcloud or third_party/datastore or something?

third_party/install.sh is a really generic path, it could install anything.